resources/assets/js/components/Dashboard.vue

```diff
@@ -0,0 +1,152 @@
+
+                                <div class="col-md-6">
+                                    <input type="text" class="form-control" name="name" v-model="addConferenceForm.name">
+                                </div>
+                            </div>
+
+                            <div class="form-group">
+                                <div class="col-md-6 col-md-offset-3">
+                                    <button type="submit" class="btn btn-primary"
+                                        @click.prevent="addConference"
+                                        :disabled="addConferenceForm.adding">
+
+                                        <span v-if="addConferenceForm.adding">
+                                            <i class="fa fa-btn fa-spinner fa-spin"></i>Adding
+                                        </span>
+
+                                        <span v-else>
+                                            <i class="fa fa-btn fa-plus"></i>Add Conference
+                                        </span>
+                                    </button>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        data: function() {
+            return {
+                currentUserId: Confomo.userId,
+
+                conferences: [],
+
+                addConferenceForm: {
+                    conference: '',
+                    errors: [],
+                    adding: false
+                },
+            };
+        },
+
+        ready: function () {
+            this.getAllConferences();
+        },
+
+        computed: {
+        },
+
+        methods: {
+            getAllConferences: function () {
+                this.$http.get('/api/conferences')
+                    .success(function (conferences) {
+                        this.conferences = conferences;
+                    });
+            },
+
+            addConference: function () {
+                // @todo: Do validation in a more VueJS-y way?
+                if (this.addConferenceForm.name == '') {
+                    this.addConferenceForm.errors = [
+                        'You need to actually type something for the name.'
+                    ];
+                    this.addConferenceForm.adding = false;
+
+                    return;
+                }
+
+                this.addConferenceForm.errors = [];
+                this.addConferenceForm.adding = true;
+
+                this.$http.post('/api/conferences', this.addConferenceForm)
+                    .success(function (conference) {
+                        this.addConferenceForm.name = '';
+                        this.addConferenceForm.adding = false;
+                        // @todo: Add new conference to the list more cleanly
+                        this.getAllConferences();
```

Comment:
Ha, yah, I really didn't know what I was doing. We can just push the conference onto this.conferences; we don't have to re-get from the API.